phantazm-api/src/main/java/com/github/phantazmnetwork/api/config/KeyConfigProcessor.java

```diff
@@ -0,0 +1,30 @@
+package com.github.phantazmnetwork.api.config;
+
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.ConfigPrimitive;
+import com.github.steanky.ethylene.core.collection.ConfigNode;
+import com.github.steanky.ethylene.core.collection.LinkedConfigNode;
+import com.github.steanky.ethylene.core.processor.ConfigProcessException;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import net.kyori.adventure.key.Key;
+import org.intellij.lang.annotations.Subst;
+import org.jetbrains.annotations.NotNull;
+
+public class KeyConfigProcessor implements ConfigProcessor<Key> {
```

Comment:
if you look at the mapdata branch, i've already made a configprocessor for Key, it can probably be moved to phantazm-commons